frontend/app/work_packages/config/index.js

```diff
@@ -52,6 +52,7 @@ angular.module('openproject.workPackages.config')
   watcher_id: {type: 'list_model', modelName: 'user', order: 6, locale_name: 'watcher'},
   responsible_id: {type: 'list_optional', modelName: 'user', order: 6, locale_name: 'responsible'},
   fixed_version_id: {type: 'list_optional', modelName: 'version', order: 7, locale_name: 'fixed_version'},
+  category_id: { type: 'list_optional', modelName: 'category', order: 7, locale_name: 'category' },
```

Comment:
Identifier 'category_id' is not in camel case.<br>Identifier 'locale_name' is not in camel case.